src/views/contact.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Container } from '../components/layout/container';
+import { Heading } from '../components/typography';
+
+export const Contact = () => {
+    return (
+        <Container>
+            <Heading>Get in touch</Heading>
+            <div>For support or any help, please reach out to <a href="mailto:helx@lists.renci.org">helx@lists.renci.org.</a></div>
```

Comment:
Just a thought, does Hannah want this as a `mailto` since people didn't like the native app launch?